core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -81,12 +81,12 @@ export class Jazz {
   }
   getProdName(){
     return element(by.xpath('//div[@class="stage-title stageDisp" and contains(text(),"prod")]'))
-  }
+	}
   getProdHeader(){
     return element(by.xpath('//div[@class="servie-details-container"]/h1[@class="page-hdr bold-title relative" and contains(text(),"prod")]'));
   }
   getAsset(){
-    return element(by.xpath('//div/div[2]/div/section[1]/div/div[2]/tabs/div/ul/li[3]'));
+    return element(by.xpath('//li[@class="x caps" and contains(text(),"assets")]'));
```

Comment:
good change. Will save us having to fix the tests even if tabs are added to removed in the future